File: src/JavaScript/RGB-To-Hex-Conversion.js

convert2hex = integer =>
  Math.max(0, Math.min(255, integer)).toString(16).padStart(2, "0");

function rgb(r, g, b) {
  return [r, g, b].map(convert2hex).join("").toUpperCase();
}

console.log(rgb(-20, 275, 125));

File: src/Julia/RGB-To-Hex-Conversion.jl

function convert2hex(x::Int64)::String
    string(clamp(x, 0, 255); base=16, pad=2)
end
convert2hex(12)


function rgb(r::Int64, g::Int64, b::Int64)::String
    [r, g, b] .|> convert2hex |> join |> uppercase
    # string(convert2hex.([r, g, b])...) |> uppercase
end
rgb(-20, 275, 125)
